### ISSUE TITLE
bogue: Utilisation du tag <article> dans les composants

### ISSUE THREAD

**Issue by SuperVignola** [ulaval/modul#373]
**Description du bogue / Describe the bug**
Certains composants modUL utilisent le tag <article> même si le contenu ne représente pas un « article ». The article role denotes a section of a document, page, or site that, if it were standing on its own, could be viewed as a complete document, page or site.

Voici la liste des composants utilisant le tag article : 
 	- accordion
	- accordion-group
	- dialog
	- message-page
	- modal
	- panel
	- popper
	- sitebar
	- card

**Comment by chuckmah** [comment]
Je suis d'accord c'est surement une mauvaise utilisation du tag article dans MODUL

@jipigi @raphpare  des idées ?

**Comment by SuperVignola** [comment]
@raphpare : je l'ai d'abord remarqué dans la table des matières d'un site de formation dans Brio. La table des matières, c'est le m-sidebar? Il me semble que le contenu de la table des matières ne correspond pas vraiment à la description du tag article.

**Comment by jipigi** [comment]
Mon avis là dessus est qu'il est difficile dans MODUL de prévoir quel type de contenu sera placé dans les composants qu'on propose, et donc l'emploi des balises sémantiques de ce type est délicat. Dans certains cas, la balise va être pertinente mais dans d'autres probablement pas. Je pense que je laisserais le soin aux projets de gérer ça. C'est du cas par cas, mais pour la balise article, c'est ma recommandation.

Techniquement, il y a deux attributs (`aria-posinset` et `aria-setsize`) qui peuvent compléter la définition de a balise articles, est-ce qu'il est possible de les définir si la balise est déjà placée à l'intérieur du composant?

**Comment by raphpare** [comment]
Pour les composants **dialog**, **modal**, **sidebar** et **mesage-page**, la balise devrait toujours être **<article>**. @jipigi, il serait possible d'ajouter des props pour les attribut **aria-posinset** et **aria-setsize** sur les composant en question.

**Comment by jipigi** [comment]
@raphpare, non ce n'est pas nécessaire d'ajouter ces propriétés pour le moment, merci.
